File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        PriorityQueue<int,int> res=new PriorityQueue<int,int>();
        foreach(int num in nums){
            res.Enqueue(num,num);
            if(res.Count>k){
                res.Dequeue();
            }
        }
        return res.Peek();
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> res=new PriorityQueue<>();
        for(int num:nums){
            res.add(num);
            if(res.size()>k){
                res.poll();
            }
        }
        return res.peek();
    }
}